concent_api/core/validation.py

```diff
@@ -373,3 +369,36 @@ def validate_file_transfer_token(file_transfer_token: message.concents.FileTrans
     categories = [file_info['category'] for file_info in file_transfer_token.files]
     if len(set(categories)) != len(categories):
         raise FileTransferTokenError("'category' field must be unique across FileInfo list.", ErrorCode.MESSAGE_FILES_CATEGORY_NOT_UNIQUE)
+
+
+def validate_secure_hash_algorithm(checksum: str):
```

Comment:
please add unit tests for the validation

Reply:
Done